SetupDataPkg/Tools/GenNCCfgData.py

```diff
@@ -231,6 +231,7 @@ def handler(id, value):
                 bin_data = base64.b64decode(base64_val)
                 var_list = read_vlist_from_buffer(bin_data)
                 actual_var_list = self.get_var_list_for_instance(var_list)
+                print("OSDDEBUG actual_var_list: ", actual_var_list)

```

Comment:
Was this line meant to be checked in?

Reply:
Absolutely not and thereby proving the value of code reviews :). I'll remove it, thanks.